moving all developer-focused docs into EMOD code repo

JSchripsema-IDM opened the pull request:
This is currently building without error, but only contains links back to the generic EMOD docs. Links to the HIV and malaria docs are in comments until they can be merged into emodpy-hiv and emodpy-malaria. Once those are merged, I can update those commented links and this PR can be reviewed.

Bridenbecker reviewed (changes requested):
There are just a couple of small things to clean up.  Thank you for saving these docs.  I did not know that some of it was even there.

JSchripsema-IDM commented on docs/conf.py:
```diff
@@ -0,0 +1,452 @@
+                       'emodpy_generic': ('https://docs.idmod.org/projects/emodpy-generic/en/latest/', None),
+                       'idmtools': ('https://docs.idmod.org/projects/idmtools/en/latest/', None),
+                       'calibra': ('https://docs.idmod.org/projects/idmtools_calibra/en/latest/', None),
+                       'emod-hiv': ('https://docs.idmod.org/projects/emod-hiv/en/latest/', None),
```
I just missed it, removed now.

JSchripsema-IDM commented on docs/_templates/footer_start.html:
```diff
@@ -0,0 +1,3 @@
+<p>emodpy-hiv is licensed under the <a href="https://creativecommons.org/licenses/by-sa/4.0/legalcode">Creative Commons
+    Attribution-ShareAlike 4.0 International License</a>.</p>
+<p>Send documentation feedback to <a href="mailto:idmsupport@gatesfoundation.org" target="_top">idmsupport@gatesfoundation.org</a>.</p>
```
Fixed this, docset name, and license.

JSchripsema-IDM commented on README.md:
```diff
@@ -85,7 +86,7 @@ https://docs.idmod.org/projects/emod-malaria/en/latest/dev-install-overview.html
 Contributing
 ------------
-Questions or comments can be directed to [support@idmod.org](<mailto:support@idmod.org>).
+Questions or comments can be directed to [support@idmod.org](<mailto:idm@gatesfoundation.org>).
```
Just a mistake on my part!

JSchripsema-IDM commented:
@Bridenbecker Thanks Dan for the thorough review! I addressed all of your comments. Once all three related EMOD PRs are merged, I'll update all the links that were commented out.

Bridenbecker commented:
@JSchripsema-IDM  FYI.  I'm assuming that **you** will merge when you think it is ready.
